lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'r',
+    required: false
+  },
+  'node-version': {
```

Comment:
Offline conversation resulted in a change to the [DD](https://docs.google.com/document/d/1acE3hyUnDRNuGg3a7eaM-kPkffRSqqdvJZvr4NLMYyQ/edit#) from `runtime-version` to `node-vesrion`.

Also, this command is `deploynodeapp` so there should never be another Turbo supported runtime deployed through this, hence the name `node-version`